tests/features/search/search_in_data_entity.spec.ts

```diff
@@ -0,0 +1,244 @@
+import { expect } from '@playwright/test';
+import { test } from '../../config/test-base';
+
+const entityNameWithAlphabeticChars = 'Book_ETL_aqa';
+const entityWithStructure = 'books_aqa';
+const entityNameWithNumbers = '737boeing';
+const entityNameBeforeRename = 'postcards_aqa';
+const entityNameAfterRename = 'renamed_data_entity';
+const entityNameBeforeRename2 = 'magazines_aqa';
+const entityNameAfterRename2 = 'renamed_data_entity2';
+const externalDescription = 'the newest books in the city';
+
+
+test.describe('Search by data in data-entity in the Catalog', () => {
+    test.beforeEach(async ({steps: {pages}, page}) => {
+        await test.step(`I open Catalog page`, async () => {
+            await page.goto('');
+            await pages.topPanel.clickTab('Catalog');
+        });
+    });
+    /**
+     * /project/1/test-cases/111
+     */
+    test(`Display an expected item with expression in internal description`, async ({steps: {pages}}) => {
```

Comment:
Looks like the prettier code formatter was not able to fix code style issues when you did your comments.

Could you please manually run `npm run lint:fix` and commit changes to fix code style issues introduced in your PR